src/main/java/org/osiam/resources/exception/OsiamExceptionHandler.java

```diff
@@ -62,10 +63,10 @@ public ErrorResponse handleResourceExists(ResourceExistsException e) {
         return produceErrorResponse(e.getMessage(), HttpStatus.CONFLICT);
     }
 
-    @ExceptionHandler(OsiamBackendFailureException.class)
+    @ExceptionHandler({OsiamBackendFailureException.class, ConnectionInitializationException.class})
```

Comment:
As a note to your original question: The `ConnectionInitializationException` should be handled here explicitly, because otherwise we would just generate a 409 CONFLICT as was the case before this commit. That's not right. If the resource-server can't reach the auth-server it's a internal server error and should be communicated like that. I will update the `CHANGELOG.md` accordingly.